### ISSUE TITLE
Background service doesn't stop on closing app

### ISSUE THREAD

**Issue by georgebushfour20** [lbryio/lbry-android#1178]
l

## The Issue
I want lbry's videos to play in the background, but only when the app hasn't been 'swiped'.  However even after swiping, there is a persistent notification saying that lbry is playing in the background.

I tried to follow what was here (https://lbry.com/faq/android-basics#stoplbryservice), though I'm not sure because I don't exactly see the setting named "Keep the daemon background service running after closing the app".

### Steps to reproduce
1. Enable background playback
2. Disable background service (in the 'other' section)
3. Swipe the app

### Expected behaviour
No trace of lbry after swiping.

### Actual behaviour
Persistent/ongoing notification saying that the lbry service is playing in the background with a "stop" buttion. I have to press this everytime I close the app.

## System Configuration
- Device: Galaxy s20
- Android version: 10

## Screenshots

(Will add, give me some time)

### Acceptance Criteria
The persistent notification shouldn't show. Disabling notifications is not an option.

### Definition of Done
- [ ]  Tested against acceptance criteria
- [ ] Tested against the assumptions of the user story
- [ ] The project builds without errors
- [ ] Unit tests are written and passing
- [ ] Tests on devices/browsers listed in the issue have passed
- [ ] QA performed & issues resolved
- [ ] Refactoring completed
- [ ] Any configuration or build changes documented
- [ ] Documentation updated
- [ ] Peer Code Review performed

**Comment by georgebushfour20** [comment]
You mean nothing runs after the app is swiped?

**Comment by kekkyojin** [comment]
No, I don't mean it. Swipping or closing any app means "close the app". If the service used was a background one, then service would also be closed. But, as it is a foreground one, although the app itself is closed -no longer in memory- the service is still running.
